html-templates/app/SlateDemonstrationsStudent/ext.tpl

```diff
@@ -22,32 +22,34 @@
     {$allContentAreas = Slate\CBL\ContentArea::getAll()}
 
     <div class="wrapper site">
-        <header class="page-header">
-            <h2 class="header-title">{$Student->FullName|escape}</h2>
-        </header>
-
-        <form method="GET">
-            {if $.get.student}
-                <input type="hidden" name="student" value="{refill student}">
-            {/if}
-
-            <div class="inline-fields">
-                <div class="field">
-                    <select class="field-control" name="content-area">
-                        {if !$ContentArea}
-                            <option value="">Select a rubric</option>
-                        {/if}
-                        {foreach item=availableArea from=$allContentAreas}
-                            <option value="{$availableArea->Code}" {refill field=content-area selected=$availableArea->Code}>{$availableArea->Title|escape}</option>
-                        {/foreach}
-                    </select>
-                </div>
-                <button class="button primary">Refresh</button>
-                {if Slate\CBL\CBL::$continuaUrl}
-                    <a class="button" href="{Slate\CBL\CBL::$continuaUrl|escape}" target="_blank">View the Continua</a>
+        <div class="inner" style="margin-bottom: -7em"> {* HACK *}
```

Comment:
This .inner wrapper is key. The diff implies a lot more changes than I really made.